libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3257,6 +3257,42 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
     ],
     explorerId: "zec",
   },
+  zcash_dev: {
+    type: "CryptoCurrency",
+    id: "zcash_dev",
```

Comment:
You may want this, so this network is hidden by default
```suggestion
    id: "zcash_dev",
    isTestnetFor: "zcash",
```